v4-client-py/v4_client_py/chain/aerial/config.py

```diff
@@ -81,7 +81,7 @@ def fetch_dydx_testnet(cls) -> "NetworkConfig":
             chain_id="dydx",
             url="grpc+https://v4.testnet.dydx.exchange",
             fee_minimum_gas_price=5000000000,
```

Comment:
I just noticed that we have this config as well. Can you update https://github.com/dydxprotocol/v4-clients/blob/main/v4-client-py/v4_client_py/chain/aerial/client/utils.py#L21 so that the `fee` param is Optional and if it's None, to use this value instead?

Reply:
done